src/components/CommentsPage.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react"
+import { fetchCommentsByArticleId } from "./api"
+import DisplayBox from "./DisplayBox"
+import CommentCard from "./CommentCard"
+import { useParams } from "react-router"
+
+function CommentsPage({article}) {
+    const [articleComments, setArticleComments] = useState([])
+    const {article_id} = useParams()
+
+    useEffect(() => {
+        fetchCommentsByArticleId(article_id).then((res) => {
+            setArticleComments(res.data.comments)
+        })
+    }, [article_id])
+
+    return (
+        <section id="vertical-list">
+            <h2>Comments for </h2>
```

Comment:
this currently doesn't give any indication as to what the comment is for, I would almost certainly recommend taking this component and rendering it on the individual article page